msgpack/ext.py

```diff
@@ -5,17 +5,10 @@
 import struct
 
 
-PY2 = sys.version_info[0] == 2
-
-if PY2:
-    int_types = (int, long)
-    _utc = None
-else:
-    int_types = int
-    try:
-        _utc = datetime.timezone.utc
-    except AttributeError:
-        _utc = datetime.timezone(datetime.timedelta(0))
+try:
+    _utc = datetime.timezone.utc
```

Comment:
This is present since at least python 3.4:

https://docs.python.org/3.4/library/datetime.html#datetime.timezone.utc

Reply:
I pushed a new commit to remove the try/except clause.